pkg/api/handlers/generic/containers_create.go

```diff
@@ -40,7 +40,9 @@ func CreateContainer(w http.ResponseWriter, r *http.Request) {
 		utils.Error(w, "Something went wrong.", http.StatusInternalServerError, errors.Wrap(err, "Decode()"))
 		return
 	}
-
+	if len(input.HostConfig.Links) > 0 {
+		utils.Error(w, utils.ErrLinkNotSupport.Error(), http.StatusBadRequest, errors.Wrapf(utils.ErrLinkNotSupport, "bad parameter"))
```

Comment:
We should only error if Links is given and true.

Reply:
links are not bool ... they are []string so in this case, i think checking the len is ok?

Reply:
Ah, other places in API link is boolean.  Sorry for confusion.